tasks/managed/push-artifacts-to-cdn/tests/mocks.sh

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env bash
+set -ex
+
+# mocks to be injected into task step scripts
+
+function internal-request2() {
```

Comment:
What's this for? Maybe just something for your testing and should be removed?

Reply:
removed